src/main/java/org/broadinstitute/hellbender/tools/dataflow/MarkDuplicatesDataflow.java

```diff
@@ -71,7 +90,11 @@ protected void setupPipeline(final Pipeline pipeline) {
         final PCollectionView<SAMFileHeader> headerPcolView = pipeline.apply(Create.of(header)).apply(View.<SAMFileHeader>asSingleton());
 
         final PCollection<GATKRead> preads = readsSource.getReadPCollection(intervals);
-        final PCollection<GATKRead> results = preads.apply(new MarkDuplicatesDataflowTransform(headerPcolView));
+
+        final OpticalDuplicateFinder finder = READ_NAME_REGEX != null ?
```

Comment:
nevermind. it works - it's in the setupPipeline. Silly github source hiding fooled me